accounts/signals.py

```diff
@@ -1,8 +1,59 @@
+import requests
 from django.apps import apps
 
 from project import settings
 
 
+def post_migrate_refresh_flatpages(sender, **kwargs):
+
+    FLATPAGELIST = {
+        "resources": "/resources/",
+        "termsofuse": "/termsofuse",
+    }
+
+    # Check first site
+    Site = apps.get_model("sites.Site")
+    site = Site.objects.first()
+
+    site.domain = settings.SITE_DOMAIN
+    site.name = settings.SITE_NAME
+
+    site.save()
+    FlatPage = apps.get_model("flatpages.FlatPage")
+
+    # Path to raw github files. English in this folder, others in locale/[langcode]/
+    github_root = "https://github.com/manybabies/mbah-research-resources/raw/master/lookit%20static%20pages"
+
```

Comment:
It looks like we translating the flatpages to static pages and then hosting them on github.

If this is true, then why don't we just remove the concept of flatpages all together and replace them with templates.  This would allow the current translation process of using `.po` files to continue and we wouldn't have to go through MBAH to make updates.